cellpack/autopack/Compartment.py

```diff
@@ -247,28 +241,31 @@ def buildMesh(self, data, mesh_store):
         self.ref_obj = self.name
         return geom
 
-    def initialize_mesh(self, mesh_store):
-        if self.filename is None:
-            return None
+    def _get_volume(self, mesh_store):
+        if self.is_sphere:
+            return 4 * math.pi * self.radius**2
```

Comment:
Is this getting the volume or the surface area? I think volume should be:
```suggestion
            return 4 * math.pi * self.radius ** 3 / 3
```